### PR TITLE
Dashboard bug fixes

### PR DESCRIPTION
**Pull request by A-Wheeto**
## Status

* Current Status: Ready for (front-end / tech) review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2911

## Review progress:

- [ ] Browser tested
- [ ] Front-end review completed
- [ ] Tech review completed

## What's changed?
- Fixed margin on ncce white button
- Fixed logic on enrolment text dependant on enrolment status
- Fix to pin icon clipping

### CODE REVIEW

File: app/components/dashboard_course_component/dashboard_course_component.html.erb [hunk 1]
```diff
@@ -30,10 +30,7 @@
     </div>
     <div class="dashboard-course-component__format">
       <div class="dashboard-course-component__format--icon">
-        <span class="govuk-body-s <%= course_icon_class(@achievement.activity) %>"></span>
-      </div>
-      <div class="dashboard-course-component__format--text">
```
Comment (msquance-stem) [inline app/components/dashboard_course_component/dashboard_course_component.html.erb:35]: Do we no longer need the format--text class? If so, do we want the modifier --icon on the class, and could it just be a format div?

**Review by msquance-stem — APPROVED** [review]
LGTM - There is a little bit of HTML that might be able to be simplified but I will leave that up to you

File: app/components/dashboard_course_component/dashboard_course_component.html.erb (hunk 1 above)
Comment (A-Wheeto) [reply app/components/dashboard_course_component/dashboard_course_component.html.erb:35]: Ah yes, I'll reduce this down to just a format div

**Review by msquance-stem — APPROVED** [review]
LGTM

**Pull request merged**